jsapp/js/components/library/assetsTableRow.es6

```diff
@@ -29,8 +24,7 @@ class AssetsTableRow extends React.Component {
     if (
       this.props.asset.asset_type !== ASSET_TYPES.collection.id &&
       this.props.asset.summary &&
-      this.props.asset.summary.row_count &&
-      this.props.asset.summary.row_count >= 2
+      this.props.asset.summary.row_count
```

Comment:
2 lines could be simplified to: `this.props.asset.summary?.row_count`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
It's first time I see this - thanks! Reminds me of coffee script :)